web/src/components/atomic/Button.vue

```diff
@@ -2,7 +2,7 @@
   <component
     :is="to === undefined ? 'button' : httpLink ? 'a' : 'router-link'"
     v-bind="btnAttrs"
-    class="relative flex flex-shrink-0 cursor-pointer items-center overflow-hidden whitespace-nowrap rounded-md border px-2 py-1 shadow-sm transition-all duration-150 disabled:cursor-not-allowed disabled:opacity-50"
+    class="relative flex flex-shrink-0 cursor-pointer items-center overflow-hidden whitespace-nowrap text-wrap rounded-md border px-2 py-1 shadow-sm transition-all duration-150 disabled:cursor-not-allowed disabled:opacity-50"
```

Comment:
`whitespace-nowrap text-wrap` should not be used together. I would also not break text in all buttons by default. Can we just allow text wrap on the login screen buttons?

Reply:
I would suggest to use `class="!whitespace-normal"` on the `Button` element in web/src/views/Login.vue instead.

Reply:
Thanks, a good solution!